crate_universe/src/splicing/splicer.rs

```diff
@@ -471,6 +484,34 @@ impl<'a> SplicerKind<'a> {
 
         Ok(())
     }
+
+    fn inject_patches(manifest: &mut Manifest, patches: &cargo_toml::PatchSet) -> Result<()> {
+        for (registry, new_patches) in patches.iter() {
+            // If there is an existing patch entry it will need to be merged
+            if let Some(existing_patches) = manifest.patch.get_mut(registry) {
+                // Error out if there are duplicate patches
+                existing_patches.extend(
+                    new_patches
+                        .iter()
+                        .map(|(pkg, info)| {
+                            if existing_patches.contains_key(pkg) {
```

Comment:
It'd be nice to not error if they patches are identical

Reply:
Good idea, done!